modules/ROOT/examples/live-demos/a11ychecker/index.html

```diff
@@ -3,7 +3,7 @@
   <p>Note: This demo has been crafted to intentionally trigger most of our accessibility rules for demonstration purposes.</p>
   <hr />
   <h2><a id="example"></a>The Tiny Logo</h2>
-  <p><a href="https://www.tiny.cloud/docs/plugins/premium/a11ychecker/#liveexample" target="_blank" rel="noopener"><img src="<p><img style="display: block; margin-left: auto; margin-right: auto;" title="Tiny Logo" src="https://www.tiny.cloud/docs/images/logos/android-chrome-256x256.png" alt="TinyMCE Logo" width="128" height="128" /></p>" alt="" width="128" height="128" /></a><a href="https://www.tiny.cloud/docs/plugins/premium/a11ychecker/#liveexample" target="_blank" rel="noopener">This is a link to this demo. The same link has been added to logo to the left.</a></p>
+  <p><a href="https://www.tiny.cloud/docs/plugins/premium/a11ychecker/#liveexample" target="_blank" rel="noopener"><img src="{{logofordemos}}" alt="" width="128" height="128" /></a><a href="https://www.tiny.cloud/docs/plugins/premium/a11ychecker/#liveexample" target="_blank" rel="noopener">This is a link to this demo. The same link has been added to logo to the left.</a></p>
```

Comment:
We'd need to update the url slug to reflect the new docs path
ie:
https://www.tiny.cloud/docs/tinymce/6/a11ychecker.html#_interactive_example

Reply:
This is all really out of scope for this work tbh, as this was just about getting the styling working for the demos not about updating the content. Additionally, we already have TINY-8302 to fix the demo content.